src/test/scala/edu/ie3/simona/model/participant/PvModelSpec.scala

```diff
@@ -0,0 +1,802 @@
+        (
+          "2019-01-01T12:00:00+01:00[Europe/Berlin]",
+          0d,
+          0.16d,
+          -0.11390730226687622d,
+        ), // 12:00
+        (
+          "2019-01-01T14:00:00+01:00[Europe/Berlin]",
+          0d,
+          0.16d,
+          0.40969147333142264d,
+        ), // 14:00
+        (
+          "2019-01-01T17:30:00+01:00[Europe/Berlin]",
+          0d,
+          0.16d,
+          1.3259893306284447d,
+        ), // 17:30
+        (
+          "2019-03-21T05:00:00+01:00[Europe/Berlin]",
+          1.359922299362157d,
+          0.16d,
+          -1.9677750757840207d,
+        ), // different j (different date)
+        (
+          "2019-01-01T05:00:00+01:00[Europe/Berlin]",
+          0d,
+          0.175d,
+          -1.9315030168609224d,
+        ), // different long, ~10°E
+        (
+          "2011-06-21T11:00:00+02:00[Europe/Berlin]",
+          2.9436292808978d,
+          0.2337d,
+          -0.2960273936975511d,
+        ), // long of Berlin (13.39E)
+        (
+          "2011-06-21T12:00:00+02:00[Europe/Berlin]",
+          2.9436292808978d,
+          0.2337d,
+          -0.034228005898401644d,
+        ), // long of Berlin (13.39E)
+        (
+          "2011-06-21T13:00:00+02:00[Europe/Berlin]",
+          2.9436292808978d,
+          0.2337d,
+          0.2275713819007478d,
+        ), // long of Berlin (13.39E)
+        (
+          "2011-06-21T14:00:00+02:00[Europe/Berlin]",
+          2.9436292808978d,
+          0.2337d,
+          0.4893707696998972d,
+        ), // long of Berlin (13.39E)
+        (
+          "2011-06-21T15:00:00+02:00[Europe/Berlin]",
+          2.9436292808978d,
+          0.2337d,
+          0.7511701574990467d,
+        ), // long of Berlin (13.39E)
+        (
+          "2011-04-05T16:00:00+02:00[Europe/Berlin]",
+          1.6181353941777565d,
+          0.2337d,
+          1.0062695999127786d,
+        ), // long of Berlin (13.39E)
+        (
+          "2011-06-21T12:00:00+02:00[Europe/Berlin]",
+          2.9436292808978d,
+          0.5449d,
+          0.2769719941015987d,
+        ), // long of Cairo (31.22E)
+      )
+
+      forAll(testCases) { (time, j, longitude, omegaSol) =>
+        // When
+        val omegaCalc = pvModel.calcHourAngleOmega(
+          ZonedDateTime.parse(time),
+          Radians(j),
+          Radians(longitude),
+        )
+
+        // Then
+        omegaCalc.toRadians shouldEqual omegaSol +- 1e-10
+      }
+    }
+
+    "calculate the sunset angle omegaSS correctly" in {
+      val testCases = Table(
+        ("latitude", "delta", "omegaSSSol"),
+        (0.9d, -0.402449d, 1.0045975406286176d), // lat: ~51.57°N
+        (0.935d, -0.402449d, 0.956011693657339d), // different lat: ~53.57°N
+        (0.9d, 0.017908d, 1.5933675693198284d), // different delta
+        (
+          0.157952297d,
+          0.384670567d,
+          1.635323424114512d,
+        ), // Example 2.2 Goswami Priciples of Solar Engineering
```

Comment:
This is supposed to be `Principles` I suppose (was a typo before, could you fix nonetheless?) Same at multiple places in this test 😆